<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAccountInfo</class>
 <widget class="PageWithUserBar" name="PageAccountInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tapahtumat</string>
  </property>
  <layout class="QVBoxLayout" name="widgetRootLayout">
   <item>
    <widget class="QLabel" name="lblAccount">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tapahtumat tilille %1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="transactionLayout">
     <item>
      <widget class="QTableView" name="tblTransactions"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="transactionBtnLayout">
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edellinen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrev">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Seuraava</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
